tests/features/class-wc-tests-onboarding.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Onboarding Themes Tests.
+ *
+ * @package WooCommerce\Tests\Onboarding-themes
+ */
+
+use \Automattic\WooCommerce\Admin\Features\Onboarding;
+
+/**
+ * Class WC_Tests_Onboarding
+ */
+class WC_Tests_Onboarding extends WC_Unit_Test_Case {
+
+	/**
+	 * Tests that the themes returned from woocommerce.com are sorted properly.
+	 * - The first result should be the active theme.
+	 * - The second result should always be Storefront.
+	 */
+	public function test_get_themes_storefront_first() {
+		// TODO: Perhaps we should mock the call to the remote in get_themes to speed things up.
```

Comment:
Rather than call this whole method (with an HTTP request flying), how about we refactor the call to `usort()` with a new public static method on the `Onboarding` class that can be unit tested directly?